collections/Add_A/MS_Add_A_188.xml

```diff
@@ -53,122 +57,180 @@
+                     <msItem>
+                        <locus>(fols. 1r–136v)</locus>
+                        <title type="desc">Temporale</title>
+                        <rubric>Dominica prima de adventu domini</rubric>
+                        <incipit>Antiphona. Ad te levavi animam. Ad Romanos. Fratres. Scientes quia hora est</incipit>
+                        <note>The Passion narratives with superscript letters: t = Christ, c = Narrator, a = the Jews (cf. K. Young, ‘Observations on the origin of the mediæval Passion-play’, <title>PMLA</title>, 25.2 (1910), 309–54 at 316, cites other Bodleian MSS. with this combination of letters, but not the present MS.): Matthew (fols. 54r–58v), Mark (fols. 60r–63v), Luke (fols. 64v–68r), and John (fols. 69v–72r).</note>
+                        <note>The Exultet given in full, noted (fols. 72v–73v).</note>
+                        <note>With the complete baptismal rite on Holy Saturday (fol. 75v–81v), ending with a short litany. Next to the rubric ‘Hic ponantur cerei ardentes in fontem &amp; insuffletur in eundem fontem ad similitudinem huius figurę’ is a large marginal Greek letter psi: Ψ (fol. 80r).</note>
+                        <note>The volume has been partly adapted into a noted missal by near-contemporary marginal additions from Easter to the 11th Sunday after Pentecost, but coloured initials and musical notation are often lacking (fols. 82v–117v).</note>
+                        <note>The litany (fol. 81r-v) has Severinus (venerated at Cologne and elsewhere) second among the confessors, after Martin.</note>
+                        </msItem>
+                     <msItem>
+                        <locus>(fols. 137r–156v)</locus>
+                        <title type="desc">Sanctorale, from St. Lucy to St. Nicholas (13 December – 6 December)</title>
+                        <rubric><locus>(fol. 136v)</locus> In natale sanctę Lucię virginis</rubric>
+                        <incipit>Antiphona. Dilexi iusticiam. Epistola. Qui gloriatur in domino glorietur </incipit>
+                        <note>
+                           The feast of the Purification (2 February) with the blessing, lighting, and processing of the candles (fols. 138r–139v).</note>
+                        <note>St Severinus’s 23 October feast appears at fol. 153r–v.</note>
+                       
+                     </msItem> <msItem>
+                           <locus>(fols. 156v–167v) </locus>
+                           <title type="desc">Common of the saints</title>
+                        </msItem>
+                     <msItem>
+                        <locus>(fol. 167v)</locus>
+                        <note>Miscellaneous additions:</note>
+                        <msItem>
+                           <note>Two Evangeliary readings, 13th-century: <q>In illo tempore. Stabat iuxta crucem Ihesu …</q> (John 19:25–27); <q>In illo tempore. Factum est dum loqueretur Ihesus ad turbas …</q> (Luke 11:27–28)</note>
```

Comment:
Probably both Marian additions; I haven't found the first, but the second is a standard reading for the eve of the Assumption (14 August).